packages/peregrine/lib/talons/CheckoutPage/useCheckoutPage.js

```diff
@@ -19,14 +24,16 @@ export const CHECKOUT_STEP = {
 };
 
 export const useCheckoutPage = props => {
```

Comment:
I'm seeing an error when I go to the checkout page after adding a single item to my cart:

[![Image from Gyazo](https://i.gyazo.com/3cc5ee3071a560ff459a3201e682e678.png)](https://gyazo.com/3cc5ee3071a560ff459a3201e682e678)

I believe it is due to not defaulting this to an empty object. Please make the below change:

```suggestion
export const useCheckoutPage = (props = {}) => {
```